app/mailers/user_mailer.rb

```diff
@@ -1,14 +1,14 @@
 class UserMailer < ApplicationMailer
   def user_membership_created_notification(user, organisation)
     notify_email to: user.email,
-                 subject: t(".subject", organisation_name: organisation.name),
-                 body: t(".body", user_name: user.full_name, organisation_name: organisation.name, service_name:, sign_in_url:)
+                 subject: t(".subject", service_name: user.service),
```

Comment:
`service_name` is already available as a helper in `ApplicationMailer`. It uses looks up the official service name from our locales.

```suggestion
                 subject: t(".subject", service_name:),
```